server/utils.js

```diff
@@ -97,3 +98,17 @@ exports.stringTemplate = (configPath, ...args) => {
 
   return ''
 }
+
+exports.imageToDataURL = (imagePath) => {
```

Comment:
What are you thoughts on updating this function so it returned a promise and used async/promise versions of the file operations so it was non-blocking in other runtime contexts? That could make it easier to decide dynamically later (in the middle of a request, for instance) to base64 encode something without locking up the main Library server.

Reply:
Will do!